docs/source/testing/testing.mdx

```diff
@@ -3,20 +3,23 @@ title: Integration testing
 description: Utilities for testing Apollo Server
 ---
 
-Apollo Server uses a [multi-step request pipeline](../integrations/plugins/#request-lifecycle-event-flow) to validate and execute incoming GraphQL operations. This pipeline supports integration with custom plugins at each step, which can affect an operation's execution. Because of this, it's important to perform **integration tests** with a variety of operations to ensure your request pipeline works as expected.
+Apollo Server uses a [multi-step request pipeline](/apollo-server/integrations/plugins/#request-lifecycle-event-flow) to validate and execute incoming GraphQL operations. This pipeline supports integration with custom plugins at each step, which can affect an operation's execution. Because of this, it's important to perform **integration tests** with a variety of operations to ensure your request pipeline works as expected.
 
 There are two main options for integration testing with Apollo Server:
-* Using `ApolloServer`'s `executeOperation` method.
-* Setting up an HTTP client to query your server.
+
+- Using `ApolloServer`'s `executeOperation` method.
+- Setting up an HTTP client to query your server.
 
 ## Testing using `executeOperation`
 
 Apollo Server's `executeOperation` method enables you to run operations through the request pipeline _without_ sending an HTTP request.
 
-The [`executeOperation` method](../api/apollo-server/#executeoperation) accepts the following arguments:
-* An object that describes the GraphQL operation to execute.
-  * This object must include a `query` field specifying the GraphQL operation to run. You can use `executeOperation` to execute both queries and mutations, but both use the `query` field.
-* An optional argument that is passed in to the `ApolloServer` instance's [`context` function](../data/resolvers/#the-context-argument).
+<!-- TODO(AS4) add link once the API article is updated -->
+The `executeOperation` method accepts the following arguments:
+
+- An object that describes the GraphQL operation to execute.
+  - This object must include a `query` field specifying the GraphQL operation to run. You can use `executeOperation` to execute both queries and mutations, but both use the `query` field.
+- An optional second object that is used as the operation's [`context`](../data/resolvers/#the-context-argument) value.
```

Comment:
i'd get rid of the typewriter text here (because this is the value, not the function) and link both words "context value"